src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -285,6 +285,7 @@
                 "downloadAllTeams": "Oder lade die Repositories aller Teams herunter",
                 "filterLateSubmissions": "Verspätete Abgaben herausfiltern",
                 "filterLateSubmissionsDate": "Datum zum Filtern verspäteter Abgaben (standardmäßig Einreichungsfrist der Aufgabe)",
+                "excludePracticeSubmissions": "Übungsmodus Abgaben ausschließen",
```

Comment:
Maybe "Abgaben aus dem Übungsmodus ausschließen"? Not sure though

Reply:
Or maybe
```suggestion
                "excludePracticeSubmissions": "Übungsmodus-Abgaben ausschließen",
```

Reply:
Yeah maybe, I was trying to keep consistency with `Verspätete Abgaben herausfiltern` (but messed up the `herausfiltern` ^^). I like your wording better. Changed.